### PR TITLE
Remove reference to version in installation scripts

### PR DESCRIPTION
**Pull request by jarhodes314**
## Proposed changes
Fix errors building main from the tests attempting to install 1.x

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [x] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [x] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by Bravo555 — APPROVED** [review]

File: ci/installation_scripts/install_for_amd64.sh [hunk 1]
```diff
@@ -14,5 +14,5 @@ sudo apt-get --assume-yes install "${EXTERNAL_AMD64_PACKAGES[@]}"
 # Install thin-edge packages
 for PACKAGE in "${RELEASE_PACKAGES[@]}"
 do
-    sudo dpkg -i ./"$PKG_DIR"/"$PACKAGE"_0.*_amd64.deb
+    sudo dpkg -i ./"$PKG_DIR"/"$PACKAGE"_1.*_amd64.deb
```
Comment (reubenmiller) [inline ci/installation_scripts/install_for_amd64.sh:17]: We could probably avoid any reference to the version, e.g. `0` or `1`...

```suggestion
    sudo dpkg -i ./"$PKG_DIR"/"$PACKAGE"_*_amd64.deb
```

Comment (jarhodes314) [reply ci/installation_scripts/install_for_amd64.sh]: Now done in both scripts

File: tests/RobotFramework/tasks/build_install_rpi.robot [hunk 2]
```diff
@@ -13,7 +13,7 @@ ${HOST}
 ${USERNAME}       pi
 ${PASSWORD}       crypt:LO3wCxZPltyviM8gEyBkRylToqtWm+hvq9mMVEPxtn0BXB65v/5wxUu7EqicpOgGhgNZVgFjY0o=    
 ${DeviceID}       
-${Version}        0.*
+${Version}        1.*
```
Comment (reubenmiller) [inline tests/RobotFramework/tasks/build_install_rpi.robot:16]: Well have to change this as well, to `*.*`

```suggestion
${Version}        *.*
```

**Review by reubenmiller — APPROVED** [review]

File: tests/RobotFramework/tests/installation/install_tedge.robot [hunk 3]
```diff
@@ -9,7 +9,7 @@ Test Teardown    Get Logs
 *** Test Cases ***
 Install latest via script (from current branch)
     Execute Command    curl -fsSL https://thin-edge.io/install.sh | sh -s
-    Tedge Version Should Match Regex    ^\\d+\\.\\d+\\.\\d+$
+    Tedge Version Should Match Regex    ^\\d+\\.\\d+\\.\\d+(-rc\\.\d+)?$
```
Comment (reubenmiller) [inline tests/RobotFramework/tests/installation/install_tedge.robot:12]: I think you're missing an escape char `\\d+` instead of `\d+`

Comment (jarhodes314) [reply tests/RobotFramework/tests/installation/install_tedge.robot]: Sod it, I'd just corrected that from when I initially missed the `\` entirely!

File: tests/RobotFramework/tests/installation/install_tedge.robot [hunk 4]
```diff
@@ -30,7 +30,7 @@ Install specific version via script (from current branch)
 
 Install latest tedge via script
     Execute Command    curl -fsSL https://thin-edge.io/install.sh | sh -s
-    Tedge Version Should Match Regex    ^\\d+\\.\\d+\\.\\d+$
+    Tedge Version Should Match Regex    ^\\d+\\.\\d+\\.\\d+(-rc\\.\d+)?$
```
Comment (reubenmiller) [inline tests/RobotFramework/tests/installation/install_tedge.robot:33]:
```suggestion
    Tedge Version Should Match Regex    ^\\d+\\.\\d+\\.\\d+(-rc\\.\\d+)?$
```

**Pull request merged**